### DISCUSSION TITLE
View/edit own records MySQL

### DISCUSSION THREAD

**Discussion by mmalessa80** [Budibase/budibase#6523] [Help]
I am referencing this topic https://github.com/Budibase/budibase/issues/3421 where @aptkingston mentions an alternate setup for SQL to use the CurrentUser.CreatedBy functionality. I have been unable to locate documentation on how to use that functionality with MySQL tables and was hoping that someone might be able to help out? I would really appreciate any assistance with this. Thank you in advance.

**Comment by melohagan** [comment]
Hi @mmalessa80 

Reading his comment he does state:
> Note that this only works for internal tables
https://github.com/Budibase/budibase/issues/3421#issuecomment-976253028

This is my understanding as well. If you want to have "Created By" columns in your MySQL tables you will have to manage that manually unfortunately!
